x-pack/plugins/maps/public/elasticsearch_geo_utils.js

```diff
@@ -60,9 +62,9 @@ function pointGeometryFactory(lat, lon) {
   };
 }
 
-export function geoPointToGeometry(value) {
+export function geoPointToGeometry(value, accumulator = []) {
   if (!value) {
-    return [];
+    return accumulator;
```

Comment:
Returns are not longer being used. Not need to return anything from this function